src/main/java/ca/bc/gov/chefs/etl/forms/pcd/chc/pcPatientServices/model/PCPSChcSubmission.java

```diff
@@ -0,0 +1,218 @@
+package ca.bc.gov.chefs.etl.forms.pcd.chc.pcPatientServices.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import ca.bc.gov.chefs.etl.constant.PCDConstants;
+import ca.bc.gov.chefs.etl.core.model.IModel;
+
+public class PCPSChcSubmission implements IModel {
+  @Override
```

Comment:
Please use the following order:
- properties
- getters/setters
- method overrides
- toString() (if used)

Reply:
Files have been refactored to follow this convention.